roles/confluent.common/tasks/ubuntu.yml

```diff
@@ -3,7 +3,9 @@
   apt_key:
     url: "{{confluent_common_repository_debian_key_url}}"
     state: present
-  when: not custom_apt_repo|bool
+  when:
+    - not custom_apt_repo|bool
```

Comment:
add same conditional to debian.yml please

Reply:
I think that conditional is there already, unless I'm missing something...

Reply:
you are right, thanks!